WFD/RBD.py

```diff
@@ -154,67 +156,60 @@ def create_new_rbd_item(self, entry_data, country):
         :param entry_data: dict with the data for the rbd
         :param country: country code
         """
-        data = {
-            'labels': {},
-            'descriptions': {}}
-
-        data['labels'] = self.make_labels(entry_data)
-        data['descriptions'] = self.make_descriptions(entry_data, country)
+        labels = WfdBot.convert_language_dict_to_json(
```

Comment:
Yes. (I've done a bunch of cleanup in #11 but forgot to add it here where it was introduced).

Fixed